doc/release-notes/9363-eln.md

```diff
@@ -0,0 +1,7 @@
+Support for .eln files has been added.
+
+See file format specification: https://github.com/TheELNConsortium/TheELNFileFormat
+
+The .eln file format is used by Electronic Laboratory Notebooks as an exchange format for experimental protocols, results, sample descriptions, etc...
+
+A Solr schema update is required.
```

Comment:
```suggestion
In order for existing .eln files to show as zip files (instead of unknown), you must run the file re-detection API on them and reindex them into Solr.
```